<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note</string>
  </property>
  <property name="styleSheet">
   <string notr="true">width:568px;
height:542px;
background-color: rgb(0, 0, 0);
border-radius:6px;
opacity:0.7;</string>
  </property>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Mini">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ukui_labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK SC</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>568</width>
     <height>397</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pushButton_Exit</zorder>
  <zorder>pushButton_Mini</zorder>
  <zorder>ukui_labelTitle</zorder>
  <zorder>toolButton</zorder>
  <zorder>listWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
